src/styles/card.scss

```diff
@@ -0,0 +1,112 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: $background;
+
+  &__stars {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin: 0;
+    padding-top: 16px;
+  }
+
+  &__img {
+    margin: 32px 19px 40px;
+    width: 160px;
+    height: 134px;
+    display: block;
+    object-fit: cover;
+  }
+
+  &__text {
+    padding-left: $padding-left;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: $line-height;
+    color: $color-text;
```

Comment:
The variable `$color-text` is used here, but it's not defined within this file. Ensure that it is defined in an imported file or within this file to avoid compilation errors.